assets/src/legacy/map.js

```diff
@@ -6421,7 +6421,15 @@ window.lizMap = function() {
         $('#headermenu .navbar-inner .nav a[rel="tooltip"]').tooltip();
         $('#mapmenu .nav a[rel="tooltip"]').tooltip();
         self.events.triggerEvent("uicreated", self);
-
+      })
+      .catch((error) => {
+        console.error(error);
+        const errorMsg = `
+          <p class="error-msg">${lizDict['startup.error']}
+          <br><a href="${lizUrls.basepath}">${lizDict['startup.goToProject']}</a></p>`;
```

Comment:
I would love to have the detail of the error messages displayed in a [<samp>](https://developer.mozilla.org/fr/docs/Web/HTML/Element/samp)  when the current authenticated user is the admin user. This would probably help the admin to understand the issue.

Hopefully it would be simple to add since you throw different messages corresponding to the failed request.